src/garage/tf/regressors/__init__.py

```diff
@@ -21,8 +17,7 @@
 from garage.tf.regressors.gaussian_mlp_regressor import GaussianMLPRegressor
 
 __all__ = [
-    'BernoulliMLPRegressor', 'BernoulliMLPRegressorWithModel',
```

Comment:
Why `BernoulliMLPRegressorWithModel` is also removed?

Reply:
Might be a mistake...